content/docs/guides/crosswalk/kubernetes/aws/01-cloud-stacks/02-networking.md

```diff
@@ -0,0 +1,130 @@
+---
+
+title: Managed Infrastructure
+
+aliases: ["/docs/guides/crosswalk/kubernetes/aws/managed-infra/"]
```

Comment:
Can you explain this alias? Did we used to have content at this URL path? (I'll stop commenting on these, but my feedback applies throughout).

Reply:
See https://github.com/pulumi/docs/pull/1850#discussion_r335169471